server/task/task-distributed/src/main/java/org/apache/james/task/eventsourcing/distributed/RabbitMQWorkQueue.java

```diff
@@ -117,15 +117,22 @@ private void consumeWorkqueue() {
         String json = new String(delivery.getBody(), StandardCharsets.UTF_8);
 
         TaskId taskId = TaskId.fromString(delivery.getProperties().getHeaders().get(TASK_ID).toString());
-
+        Task task;
         try {
-            Task task = taskSerializer.deserialize(json);
+            task = taskSerializer.deserialize(json);
+        } catch (Exception e) {
+            LOGGER.warn("Unable to deserialize submitted Task  {}", taskId.asString(), e);
```

Comment:
why is it not `error` level?

Reply:
@mbaechler maked me change it to `warn` or `info` because the error is already represented in the status of the Task.

Reply:
We updated the log level to error for the log in the case of fail in deserialization